### ISSUE TITLE
Add more columns to the player window

### ISSUE THREAD

**Issue by knochenhans** [emoon/HippoPlayer#124]
More columns in the player window would be helpful, ideally configurable via right click. Especially, a file path or file name column would be practical for debugging purposes at this point to identify specific files when loading multiple files.

**Comment by emoon** [comment]
Yeah. I have thought about this as well but had forgot to create a issue for it so thanks :)
